src/utils/formatNumbers.ts

```diff
@@ -0,0 +1,78 @@
+/* Copied from Uniswap/v-3: https://github.com/Uniswap/v3-info/blob/master/src/utils/numbers.ts */
```

Comment:
This link will fall out of date. Is it important to keep in sync? If not, you can omit the comment.

Reply:
nope, removed